aea/cli/local_registry_sync.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""Implementation of the 'aea sync-local-registry' subcommand."""
+import os
+import shutil
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Generator, Tuple, Union
+
+import click
+
+from aea.cli.fingerprint import determine_package_type_for_directory
+from aea.cli.registry.add import fetch_package
+from aea.cli.registry.utils import get_package_meta
+from aea.cli.utils.context import Context
+from aea.cli.utils.decorators import pass_ctx
+from aea.configurations.data_types import PackageId, PackageType, PublicId
+from aea.configurations.loader import load_component_configuration
+
+
+PACKAGES_DIRS = [i.to_plural() for i in PackageType]
+
+
+@click.command()
+@pass_ctx
+def local_registry_sync(ctx: Context):
+    """Upgrade the local package registry."""
+    do_local_registry_update(ctx.cwd)
```

Comment:
So this needs to be run from within `packages`?

Reply:
yes